### DISCUSSION TITLE
After OTA update Unable to access GUI

### DISCUSSION THREAD

**Discussion by shambahri** [nakhonthai/ESP32APRS_T-TWR#27] [General]
Hi Dev,
After i updated via OTA  0.3b I unable to login to main Dashboard.
Username - admin, Password - admin , seems no longer works.
thanks.

**Comment by lyusupov** [comment]
>  Username - admin, Password - admin , seems no longer works.

https://github.com/nakhonthai/ESP32APRS_T-TWR/issues/18#issuecomment-1886574522

**Reply by shambahri** [reply]
Yeah, some corruptions after the OTA, default user/password screwup. 
Thank you again for the tips gents,  
HW reset works. 
Cheers.

**Reply by dboggny** [reply]
I did a factory reset and saved config to file and it worked after that.
